### ISSUE TITLE
analysis: difference in expression with change in annotation

### ISSUE THREAD

**Issue by blahah** [COMBINE-lab/measuresmatter#11]

**Comment by rob-p** [comment]
@pimentel Yes; my reading is that in STE, the coverage of each transcript is approximately equivalent.  Which, of course, means that either (1) the transcripts are covered to approximately the same depth, and therefore, they must be present in (potentially *vastly*) different abundances (in accordance with their lengths) or (2) the number of reads deriving from each transcript is approximately equal, and, therefore, they must again be present in potentially vastly different abundances.  To me, this suggests that their "gene expression" values, even in the STE conditions, derives from a set of transcripts with potentially vastly different abundances.  Good catch on the lack of documenting how MTE actually works!

**Comment by rob-p** [comment]
I also added a reference to the [Laiho et al.*](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0115964) paper which is related  but somewhat different in aim from the Fonseca *et al.* paper.  They try to identify DE genes using an exon based approach, and they seem to contrast it against the (*wrong*) gene read-count approach.  It may be more relevant to the DE section, but I think we should also address why such strategies (i.e. counting reads on exons) are not really better than gene read-counting.
